.npmrc

```diff
@@ -1 +0,0 @@
-package-lock=false
```

Comment:
I've found that `package-lock=false` also disables `npm-shrinkwrap.json`, hence deleting this line and the resulting blank file.